algorithms/gr-dfsrh/javascript/dfs.js

```diff
@@ -0,0 +1,41 @@
+
+const dfs = (graph, startNode) => {
+    // Create an array / fake "stack" for storing next visiting nodes
+    let stack = [startNode];
+    let visited = [];
+
+    while (stack.length) {
+        // Take first node from stack to visit
+        node = stack.shift();
```

Comment:
Codacy found an issue: ['node' is not defined.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)